docs: grammatical changes to how-to-work-on-coding-challenges (#37266)
freeCodeCamp/freeCodeCamp: docs/how-to-work-on-coding-challenges.md

The 2-minute rule forces you, the challenge designer, to make your directions concise, your seed code clear, and your tests straight-forward.

> We have JavaScript events that track how long it takes for campers to solve challenges, and we can use them to identify challenges that need to be simplified or split.

Reviewer: Suggested change:
```suggestion
We track how long it takes for campers to solve changes and use this information to identify challenges that need to be simplified or split.
```
